snapcraft.yaml

```diff
@@ -0,0 +1,32 @@
+name: snapd
+summary: Daemon and tooling that enable snap packages
+description: |
+ Install, configure, refresh and remove snap packages. Snaps are
+ 'universal' packages that work across many different Linux systems,
+ enabling secure distribution of the latest apps and utilities for
+ cloud, servers, desktops and the internet of things.
+
+ This is the CLI for snapd, a background service that takes care of
```

Comment:
Nitpick: this part of the description looks a bit confusing (and yes, I know it comes from our control file, which might need re-visiting too)